### PR TITLE
Add release notes for 2.10.0.0

### PR DESCRIPTION
**Pull request by peternied**
### Description
Add release notes for 2.10.0.0

### Check List
- [ ] New functionality includes testing
- [X] New functionality has been documented
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by peternied** [comment]
Preparing the list of release notes in advance of the 2.10 release [1], if any additional features make it before code freeze they will be included.



- [1] https://github.com/opensearch-project/security-dashboards-plugin/issues/1519

File: release-notes/opensearch-security-dashboards-plugin.release-notes-2.10.0.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+* Security Roles and Audit Log Settings follow Dashboard themes ([#1558](https://github.com/opensearch-project/security-dashboards-plugin/pull/1558))
+* Support OpenSearch logo theming for light / dark modes ([#1568](https://github.com/opensearch-project/security-dashboards-plugin/pull/1568))
+
+### Bug Fixes
+* When following a link that prompts sign in through SAML or Multi-Auth the destination page is not lost ([#1557](https://github.com/opensearch-project/security-dashboards-plugin/pull/1557))
+
+
+### Maintenance
+* Force resolution of selenium-webdriver to 4.10.0 ([#1541](https://github.com/opensearch-project/security-dashboards-plugin/pull/1541))
+* Change the regex command in install dashboard GHA ([#1533](https://github.com/opensearch-project/security-dashboards-plugin/pull/1533)
```
Comment (scrawfor99) [inline release-notes/opensearch-security-dashboards-plugin.release-notes-2.10.0.0.md:16]:
```suggestion
* Change the regex command in install dashboard GHA ([#1533](https://github.com/opensearch-project/security-dashboards-plugin/pull/1533))
```

**Review by cwperks — APPROVED** [review]

**Review by RyanL1997 — APPROVED** [review]

File: release-notes/opensearch-security-dashboards-plugin.release-notes-2.10.0.0.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## 2023-08-31 Version 2.10.0.0
+
+Compatible with OpenSearch-Dashboards 2.10.0
+
+### Enhancements
+* Security Getting Started page follows Dashboard themes ([#1538](https://github.com/opensearch-project/security-dashboards-plugin/pull/1538))
+* Security Roles and Audit Log Settings follow Dashboard themes ([#1558](https://github.com/opensearch-project/security-dashboards-plugin/pull/1558))
+* Support OpenSearch logo theming for light / dark modes ([#1568](https://github.com/opensearch-project/security-dashboards-plugin/pull/1568))
```
Comment (DarshitChanpura) [inline release-notes/opensearch-security-dashboards-plugin.release-notes-2.10.0.0.md:8]: should we also include #1550 and #1559 here?

https://github.com/opensearch-project/security-dashboards-plugin/commits/2.10

Comment (DarshitChanpura) [reply release-notes/opensearch-security-dashboards-plugin.release-notes-2.10.0.0.md:8]: nvm. I see the original PRs were included here.

**Review by DarshitChanpura — APPROVED** [review]

**Comment by gaiksaya** [comment]
Hi @peternied Can we backport this to 2.10 branch?
Thanks!

**Pull request merged**